common/deities/01_canaanite.txt

```diff
@@ -173,14 +177,14 @@ deity_yam = {
 		}
 	}
 	icon = deity_cr_car_yam
-	passive_modifier = { monthly_military_experience_modifier = deity_monthly_military_experience_modifier_svalue }
+	passive_modifier = { ship_repair_at_sea = deity_ship_repair_at_sea }
```

Comment:
no repair at sea, please revert

Reply:
I didn't touch this file. How can that happen? 

I changed it, I'll try to push it again

Reply:
It might be because you haven't pull the latest code or because we weren't pointing to the right branch

Reply:
Okay I think this is all fixed now